client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import FormattedHeader from 'components/formatted-header';
+import MaterialIcon from 'components/material-icon';
+import PromoCard from 'components/promo-section/promo-card';
+import PromoCardCTA from 'components/promo-section/promo-card/cta';
+import WhatIsJetpack from 'components/jetpack/what-is-jetpack';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import useTrackCallback from 'lib/jetpack/use-track-callback';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Asset dependencies
+ */
+import JetpackBackupSVG from 'assets/images/illustrations/jetpack-backup.svg';
+import JetpackScanSVG from 'assets/images/illustrations/jetpack-scan.svg';
+
+interface Props {
+	product: 'backup' | 'scan';
+}
+
+const contentPerPrimaryProduct = {
+	backup: {
+		documentHeadTitle: 'Activate Jetpack Backup now',
+		header: translate( 'Jetpack Backup' ),
+		primaryPromo: {
+			title: translate( 'Get time travel for your site with Jetpack Backup' ),
+			image: { path: JetpackBackupSVG },
+			content: translate(
+				'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
+			),
+			promoCTA: {
+				text: translate( 'Activate Jetpack Backup now' ),
+			},
+		},
+		secondaryPromo: {
+			title: translate( 'Jetpack Scan' ),
+			icon: 'security',
+			content: translate(
+				'Automated scanning and one-click fixes to keep your site ahead of security threats.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Scan gives you automated scanning and one-click fixes to keep your site ahead of security threats.' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-business-automated-transfer)